views/about_me.erb

```diff
@@ -0,0 +1,74 @@
+<html>
+<head>
+<title>Lauren Granger's Awesome Website of Awesomeness</title>
+</head>
+
+<body>
+  <header>
+    <h1>Welcome to Lauren's Awesome Website of Awesomeness</h1>
+  <nav>
+    <ul>
+      <strong><li><a href="index">Home</a></li></strong>
+      <li><a href="about_me">All About Lauren</a></li>
+      <li><a href="blog">Blog</a></li>
+      <li><a href="projects">Projects</a></li>
+    </ul
+  </nav>
+  <img src="./images/break_line.png"/>
+  </header>
+
+  <table style="text-align: center">
+    <thead>
+      <th colspan="3"><h1>
+        Hello World! I'm Lauren!
+      </h1></th>
+    </thead>
+    <tbody>
+      <tr>
+        <td colspan="3">
+          <img src="./images/lauren.png" width="200">
+        </td>
+      </tr>
+      <tr>
+        <td colspan="3">
+          me, taking my lego-logo project way to seriously, circa 1997
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <img src="./images/food.png" width="300"/>
+        </td>
+        <td>
+          <a href="http://www.adadevelopersacademy.org"><img src="./images/ada.png"/></a>
+        </td>
+        <td>
+          <img src="./images/silly_penny.png" height="300"/>
+        </td>
+      </tr>
+      <tr>
+        <td>
+          I love to cook, a passion I picked up from a love of food, a decision to live a vegan lifestyle, and a few food allergies.
+        </td>
+        <td>
+          I am a student at Ada Developer's Academy</a>
+        </td>
+        <td>
+          Most of all I like hanging out with my dog Penny. We nap, hike, and generally goof-off. She's my #1 love!
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</body>
+
+<footer>
+  <img src="./images/break_line.png" />
+    <p>
+      "Most good programmers do programming not because they expect to get paid or get adulation by the public, but because it is fun to program." - Linus Torvalds
+    </p>
+    <p><i>follow me:
+      <a href="http://twitter.com/bumbbles21">Twitter</a>
+      <a href="http://instagram.com/pennypeachesthepit">Penny's Instagram</a>
+      <a href="http://pinterest.com/bumbbles21">Pinterest</a></i>
```

Comment:
Same comment re: having the `i` tag on a separate line to make it more clearly readable where it starts and ends